### PR TITLE
feat(pagination): Add CSS vars for $pagination

### PR DESCRIPTION
**Pull request by isabellechanclou**
Add CSS vars in Pagination component.

### CODE REVIEW

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
Many review for Bootstrap side + Boosted mods

File: scss/_pagination.scss [hunk 1]
```diff
@@ -37,18 +55,18 @@
...
   text-decoration: if($link-decoration == none, null, none);
   background-color: var(--#{$prefix}pagination-bg);
   border: var(--#{$prefix}pagination-border-width) solid var(--#{$prefix}pagination-border-color);
   @include transition($pagination-transition);
   // Boosted mod: embed icon as mask-image
   @include button-icon(
-    $pagination-icon,
-    $pagination-icon-size,
-    $size: $pagination-icon-width $pagination-icon-height,
+    var(--#{$prefix}pagination-icon), // Boosted mod
```
[the remark this answers is not in the document]
Comment (louismaximepiton) [reply scss/_pagination.scss:67]: Re reading this : Not in Bootstrap so forget about it

File: scss/_pagination.scss [hunk 2]
```diff
@@ -38,17 +56,17 @@
...
```
Comment (louismaximepiton) [inline scss/_pagination.scss:67]: Didn't see at first but inside a Boosted mod so we don't need the comments

**Review by louismaximepiton — APPROVED** [review]
Sorry, I missed some Boosted mods with first review

**Review by louismaximepiton — APPROVED** [review]
LGTM, few questions to answer before merging

**Comment by julien-deramond** [comment]
> LGTM, few questions to answer before merging

What's the status of this PR. Approved on your side @louismaximepiton?

**Comment by louismaximepiton** [comment]
Yup the PR is fine to me, but we need to propose some changes to Bs directly, since some replaced variables still exist in Bootstrap with Sass only.

File: scss/_pagination.scss [hunk 3]
```diff
@@ -4,29 +4,47 @@
   // scss-docs-start pagination-css-vars
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply scss/_pagination.scss:4]: In terms of consistency and maintenance let's keep the same order as Boostrap (you can add the Boosted specific ones wherever you want). There are no line breaks neither.

File: scss/_pagination.scss [hunk 4]
```diff
@@ -67,17 +85,17 @@
     color: var(--#{$prefix}pagination-focus-color);
     background-color: var(--#{$prefix}pagination-focus-bg);
     border-color: var(--#{$prefix}pagination-hover-border-color); // Boosted mod
-    outline: $pagination-focus-outline;
+    outline: var(--#{$prefix}pagination-focus-outline); // Boosted mod
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply scss/_pagination.scss:88]: Haven't looked at it but for this one and https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1268/files#r881440526, @isabellechanclou you could create a PR for Bootstrap to propose those modifications.

**Review by julien-deramond — CHANGES_REQUESTED** [review]
I let you modify the order of the CSS vars and remove the line breaks. I'll review it more in depth after. It'll be easier to review it like that for me.

**Review by julien-deramond — APPROVED** [review]
LGTM! Renamed `--#{$prefix}pagination-active-item-border` in `--#{$prefix}pagination-active-item-border-color` and reorder them.

Regarding new CSS vars, IMO they could be added in Bootstrap as well. However IDK if they should be all in `.pagination`, so let's try it and get some feedback.

**Pull request merged**